changelogs/fragments/bgp_global_neighbor_password.yml

```diff
@@ -0,0 +1,3 @@
+---
+-minor_changes:
+-  - Add support for setting encryption_password for BGP neighbors in bgp_global module
```

Comment:
```suggestion
minor_changes:
  - Add support for setting encryption_password for BGP neighbors in bgp_global module
```

Reply:
whoops. That's what I get for looking at `git log -p` for an example of the format